docs/framework/net-native/migrating-your-windows-store-app-to-net-native.md

```diff
@@ -151,7 +151,7 @@ The use of localized resources with the <xref:System.Diagnostics.Tracing.EventSo
 
 **Miscellaneous APIs**
 
-- The [TypeInfo.GUID](xref:System.Type.GUID) property throws a <xref:System.PlatformNotSupportedException> exception if a <xref:System.Runtime.InteropServices.GuidAttribute> attribute isn't applied to the type. The GUID is used primarily for COM support.
+- The <xref:System.Type.GUID?displayProperty=nameWithType> property throws a <xref:System.PlatformNotSupportedException> exception if a <xref:System.Runtime.InteropServices.GuidAttribute> attribute isn't applied to the type. The GUID is used primarily for COM support.
```

Comment:
I’d revert this change. The link text was purposely using a different API name since we can’t represent inherited APIs with traditional links. I’ve opened a bug for that new rule that started showing up.